content/docs/concepts/security/index.md

```diff
@@ -726,3 +751,22 @@ spec:
     kind: ServiceRole
     name: "products-viewer"
 {{< /text >}}
+
+### Using other authorization mechanisms
+
+While we strongly recommend using Istio authorization mechanisms,
+Istio is flexible enough to allow you to plug in your own authentication and authorization mechanisms via the Mixer component.
+To use and configure plugins in Mixer, visit our [policies and telemetry adapters docs](/docs/concepts/policies-and-telemetry/#adapters).
+
+## Upcoming features
+
+- Identity or CA pluggability to support bring-your-own-CA and bring-your-own-identity.
```

Comment:
or -> Istio's?

Best practices doc states we should not make forward looking statements in the documentation website.

Reply:
@wattli I took a look at other pages under concepts, and it seems they don't have the future work sections as well. Do you agree to remove this section?

Reply:
I support @sdake 
Future features should be documented as they are released, otherwise we can create confusion in readers when they try to use them and they don't work.